expertise/models/multifacet_recommender/specter.py

```diff
@@ -13,6 +13,9 @@
 import torch
 from tqdm import tqdm
 from typing import Optional
+import redisai
+import redis
```

Comment:
So we need both libraries for redisAI to work?

Reply:
I think only `redisai` should suffice, will have to check.